crates/ruff_python_formatter/src/statement/stmt_class_def.rs

```diff
@@ -33,6 +34,28 @@ impl FormatNodeRule<StmtClassDef> for FormatStmtClassDef {
 
         write!(f, [text("class"), space(), name.format()])?;
 
+        if !type_params.is_empty() {
+            let sequence_end = {
+                if !bases.is_empty() {
+                    bases.first().unwrap().start()
+                } else if !keywords.is_empty() {
+                    keywords.first().unwrap().start()
+                } else {
+                    body.first().unwrap().start()
+                }
+            };
+
+            parenthesized(
+                "[",
+                &FormatTypeParamsClause {
+                    sequence_end,
+                    type_params,
+                },
+                "]",
```

Comment:
Nit: I would move the `parenthesized` call into `FormatTypeParamsClause` because it seems to be necessary on every call site. 

We could even go one step further and also move the `is_empty` check into `FormatTypeParamsClause`.